File: app/src/main/java/com/nexaflow/saayalite/domain/model/ZoneCard.kt
package com.nexaflow.saayalite.domain.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class ZoneCard(
    @SerialName("station_id") val stationId: String,
    @SerialName("area_name") val areaName: String,
    @SerialName("full_areas") val fullAreas: String,
    @SerialName("risk_level") val riskLevel: String,
    @SerialName("risk_tier") val riskTier: String,
    @SerialName("incident_count") val incidentCount: Int,
    @SerialName("women_safety_count") val womenSafetyCount: Int,
    @SerialName("top_crimes") val topCrimes: String,
    @SerialName("risk_notes") val riskNotes: String,
    @SerialName("tourist_spots") val touristSpots: String,
)

File: src/domain/model/zoneCard.ts
/**
 * Display-ready detail for a non-safe zone.
 *
 * Text fields intentionally preserve the bundled asset verbatim: they are
 * already formatted for presentation and are not derived by the parser.
 */
export interface ZoneCard {
  readonly stationId: string;
  readonly areaName: string;
  readonly fullAreas: string;
  readonly riskLevel: string;
  readonly riskTier: string;
  readonly incidentCount: number;
  readonly womenSafetyCount: number;
  readonly topCrimes: string;
  readonly riskNotes: string;
  readonly touristSpots: string;
}
